src/data/battle_moves.h

```diff
@@ -11386,4 +12720,533 @@ const struct BattleMove gBattleMoves[MOVES_COUNT] =
+        .type = TYPE_NORMAL,
+        .accuracy = 0,
+        .pp = 1,
+        .secondaryEffectChance = 0,
+        .target = MOVE_TARGET_SELECTED,
+        .priority = 0,
+        .flags = 0,
+        .zMovePower = 0,
+        .split = SPLIT_PHYSICAL,
+        .zMoveEffect = 0
+    },
+    [MOVE_GENESIS_SUPERNOVA] =
+    {
+        .effect = EFFECT_DAMAGE_SET_TERRAIN,
+        .power = 185,
+        .type = TYPE_PSYCHIC,
+        .accuracy = 0,
+        .pp = 1,
+        .secondaryEffectChance = 0,
+        .target = MOVE_TARGET_SELECTED,
+        .priority = 0,
+        .flags = 0,
+        .zMovePower = 0,
+        .split = SPLIT_SPECIAL,
+        .argument = 0,  //psychic terrain
+        .zMoveEffect = 0
+    },
+    [MOVE_SINISTER_ARROW_RAID] =
+    {
+        .effect = EFFECT_HIT,
+        .power = 180,
+        .type = TYPE_GHOST,
+        .accuracy = 0,
+        .pp = 1,
+        .secondaryEffectChance = 0,
+        .target = MOVE_TARGET_SELECTED,
+        .priority = 0,
+        .flags = 0,
+        .zMovePower = 0,
+        .split = SPLIT_PHYSICAL,
+        .zMoveEffect = 0
+    },
+    [MOVE_MALICIOUS_MOONSAULT] =
+    {
+        .effect = EFFECT_HIT,
+        .power = 180,
+        .type = TYPE_DARK,
+        .accuracy = 0,
+        .pp = 1,
+        .secondaryEffectChance = 0,
+        .target = MOVE_TARGET_SELECTED,
+        .priority = 0,
+        .flags = 0,
+        .zMovePower = 0,
+        .split = SPLIT_PHYSICAL,
+        .zMoveEffect = 0
+    },
+    [MOVE_OCEANIC_OPERETTA] =
+    {
+        .effect = EFFECT_HIT,
+        .power = 195,
+        .type = TYPE_WATER,
+        .accuracy = 0,
+        .pp = 1,
+        .secondaryEffectChance = 0,
+        .target = MOVE_TARGET_SELECTED,
+        .priority = 0,
+        .flags = 0,
+        .zMovePower = 0,
+        .split = SPLIT_SPECIAL,
+        .zMoveEffect = 0
+    },
+    [MOVE_SPLINTERED_STORMSHARDS] =
+    {
+        .effect = EFFECT_DAMAGE_SET_TERRAIN,
+        .power = 190,
+        .type = TYPE_ROCK,
+        .accuracy = 0,
+        .pp = 1,
+        .secondaryEffectChance = 0,
+        .target = MOVE_TARGET_SELECTED,
+        .priority = 0,
+        .flags = 0,
+        .zMovePower = 0,
+        .split = SPLIT_PHYSICAL,
+        .argument = 1,  //remove terrain
+        .zMoveEffect = 0
+    },
+    [MOVE_LETS_SNUGGLE_FOREVER] =
+    {
+        .effect = EFFECT_HIT,
+        .power = 190,
+        .type = TYPE_FAIRY,
+        .accuracy = 0,
+        .pp = 1,
+        .secondaryEffectChance = 0,
+        .target = MOVE_TARGET_SELECTED,
+        .priority = 0,
+        .flags = 0,
+        .zMovePower = 0,
+        .split = SPLIT_PHYSICAL,
+        .zMoveEffect = 0
+    },
+    [MOVE_CLANGOROUS_SOULBLAZE] =
+    {
+        .effect = EFFECT_ALL_STATS_UP_HIT,
+        .power = 185,
+        .type = TYPE_DRAGON,
+        .accuracy = 0,
+        .pp = 1,
+        .secondaryEffectChance = 100,
+        .target = MOVE_TARGET_BOTH,
+        .priority = 0,
+        .flags = 0,
```

Comment:
This Z-Mve move is affected by Soundproof, so it should have the `FLAG_SOUND` set.